src/pineko/cli/check.py

```diff
@@ -36,54 +38,46 @@ def sub_compatibility(grid_path, operator_path, xif):
         rich.print("[red]Error:[/]", e)
 
 
+class Scale(Enum):
+    """Auxiliary class to list the possible scale variations."""
+
+    REN = auto()
+    FACT = auto()
+
+
 @subcommand.command("scvar")
 @click.argument("grid_path", metavar="PINEAPPL", type=click.Path(exists=True))
 @click.argument(
     "scale",
     metavar="SCALE",
-    type=click.Choice(["ren", "fact"]),
+    type=click.Choice(list(map(lambda x: x.name.upper(), Scale)), case_sensitive=False),
```

Comment:
You already wrote them uppercase, and so the `map` should be redundant:
```suggestion
    type=click.Choice(list(el.name for el in Scale), case_sensitive=False),
```

And in any case you set `case_sensitive=False`, so the rest should be up to him.